defaults/policies/advice/otel.rego

```diff
@@ -0,0 +1,97 @@
+package advice
+
+import rego.v1
+
+# Use pre-computed sets from jq
+attributes_set := data.attributes_set
+
+templates_set := data.templates_set
+
+namespaces_to_check_set := data.namespaces_to_check_set
+
+name_regex := "^[a-z][a-z0-9]*([._][a-z0-9]+)*$"
+
+derive_namespaces(name) := [
+concat(".", array.slice(parts, 0, i)) |
+	parts := split(name, ".")
+	count(parts) > 1 # Only derive namespaces if there are at least 2 parts
+
+	# Stop at count(parts)-1 to exclude the full attribute name
+	some i in numbers.range(1, count(parts) - 1)
+]
+
+# checks attribute has a namespace
```

Comment:
should we enforce this by default on non-otel projects?

We actually allow raw names for users.

I'm on the fence of whether this should be default.  I think it *is* good advice, and I like you have it as improvement.